src/utils/key.rs

```diff
@@ -99,95 +131,65 @@ impl KeyBundle {
     }
 
     pub fn sign(&self, data: &[u8]) -> Result<Vec<u8>, RvError> {
-        match self.key_type.as_str() {
-            "rsa" => {
-                let rsa = Rsa::private_key_from_pem(&self.key)?;
-                let pkey = PKey::from_rsa(rsa)?;
-                let mut signer = Signer::new(MessageDigest::sha256(), &pkey)?;
-                signer.set_rsa_padding(Padding::PKCS1)?;
-                signer.update(data)?;
-                return Ok(signer.sign_to_vec()?);
-            }
-            "ec" => {
-                let ec_key = EcKey::private_key_from_pem(&self.key)?;
-                let pkey = PKey::from_ec_key(ec_key)?;
-                let mut signer = Signer::new(MessageDigest::sha256(), &pkey)?;
-                signer.update(data)?;
-                return Ok(signer.sign_to_vec()?);
-            }
-            _ => {
-                return Err(RvError::ErrPkiKeyOperationInvalid);
-            }
+        let digest = match self.key_type.as_str() {
+            "rsa" | "ec" => MessageDigest::sha256(),
+            "sm2" => MessageDigest::sm3(),
```

Comment:
Same as above, no `#[cfg(not(tongsuo))]` needed?

Reply:
> Same as above, no `#[cfg(not(tongsuo))]` needed?

fixed.